### ISSUE TITLE
Grouping By Letter rather than in groups of 400

### ISSUE THREAD

**Issue by ckronengold** [MoleMan1024/audiowagon#53]
As an audiophile with a large library and a 512gb USB stick, it would be beneficial if a large (+400) album / artist grouping were broken down into sub-groups by letter rather than the first 400, so I can more easily navigate without having to read the grouping. 

Instead of a grouping being "Born in the USA.....Greetings from Asbury Park", I was presented with a "B", a "C" , etc...and "G" menu group. I realize that having more than 400 artists or albums within any one letter would cause a headache with this method, too, but imagine it would be an extreme use case.

**Comment by MoleMan1024** [comment]
> I realize that having more than 400 artists or albums within any one letter would cause a headache with this method, too, but imagine it would be an extreme use case.

This is exactly why I did not do it by letter. This also applies to tracks and think of all the tracks that start with "I ..." this will easily go above 400 with a large library (I just checked mine and I have around 1900). And based from what I have seen some users do with the app, my library is still quite small...

Also you will have issues with non-Latin characters using this method.

**Comment by ckronengold** [comment]
Makes sense. I can see why Tracks would cause a problem more than Artists or Albums. 
My car library is relatively small because I was primarily concerned with high-res files. Those 24-192 symphonies can be 600mb per track. 

Thanks for the consideration.

**Comment by CONSULitAS** [comment]
And how about an letter navigator like in the iOS Contacts App?

Just tap on "M" to jump to the first record starting with "M". All non ASCII characters are on "#".

![IMG_D7D4E46DE355-1](https://user-images.githubusercontent.com/6808242/175343792-e8444b22-787f-4f32-9ad3-563362727599.jpeg)

**Comment by thievingjoker** [comment]
Wondering if it would be possible to have settings to allow making lists break up with less than 400 entries. When I look at my Album and Artists, they break into 4 lists of 400 each, making things really hard to find (and strangely one ends up with massive icons and the other with small ones).  If we could set the lists to break up at less than 400 independently (say 100 for Artists, 150 for Albums) so that lists would not require so much scrolling? 
![PXL_20220903_173505766 MP](https://user-images.githubusercontent.com/100040255/188297258-7faee775-b8fd-4287-90fe-b5294955c924.jpg)
![PXL_20220903_173456850](https://user-images.githubusercontent.com/100040255/188297272-aad5f19d-cb96-42cf-91e4-ad8d8e0f7119.jpg)
